app/src/main/java/com/arctouch/movies/home/view/fragment/MoviesFragment.java

```diff
@@ -0,0 +1,127 @@
+package com.arctouch.movies.home.view.fragment;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.arctouch.movies.R;
+import com.arctouch.movies.base.view.BaseFragment;
+import com.arctouch.movies.home.view.adapter.HomeMoviesAdapter;
+import com.arctouch.movies.movie.model.Movie;
+import com.arctouch.movies.movie.view.MovieDetailsActivity;
+
+import java.lang.annotation.Retention;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public class MoviesFragment extends BaseFragment implements HomeMoviesAdapter.OnMovieItemClickListener {
+
+    @Retention(SOURCE)
+    @IntDef({TYPE_MOVIES, TYPE_FAVORITES})
+    public @interface FragmentType {
+    }
+
+    public static final int TYPE_MOVIES = 0x01;
+    public static final int TYPE_FAVORITES = 0x02;
+
+    public static final String EXTRA_TYPE = "extra_type";
+
+    private View view;
```

Comment:
Isn't this available through `getView()`? Why are you keeping it?

Reply:
Yes, it is available. I always use View attribute in fragments, I really not checked this option to get view.